src/vercel-async-store.ts

```diff
@@ -0,0 +1,48 @@
+import { IAsyncStore } from '@eppo/js-client-sdk-common';
+
+import EdgeConfigStoreService from './edge-config';
+
+export default class VercelAsyncStore<T> implements IAsyncStore<T> {
+  private expirationTimeSeconds = 20;
```

Comment:
This should be configurable as an optional argument passed in top-level; if missing can default to `30`, which is our typical polling frequency for other SDKs